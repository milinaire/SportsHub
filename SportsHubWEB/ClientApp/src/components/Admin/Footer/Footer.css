.admin-footer{
    width: 100%;
    height:900px;
    min-width: 561px;
    display: flex;
    flex-direction: column;
    margin-top: 40px;
}

.admin-footer-header{
    min-width: 561px;
    width: 40%;
    display: flex;
    justify-content: space-around;


}
.footer-wrapper{
    height:900px;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
}
.admin-footer-header>a:visited{
    color: red;
}
.footer-block{
    width:60%;
    height:400px;
    margin-top:50px;
    margin-bottom: 20px;
    min-width: 561px;
}
.footer-block1{
    width:60%;
    height:auto;
    margin-top:-200px;
    margin-bottom: 20px;
    min-width: 561px;
}
.footer-head{ background-color:rgba(242, 242, 242);
    height:60px;
    display:flex;

    align-items: center;}
.sub-footer-head1{
    height:60px;
    width:65%;
    display:flex;
    justify-content: flex-start;
    align-items: center;
    padding-left: 20px;
}
.footer-head>.sub-footer-head1>p{
    margin: 0;
    padding: 0;
    font-weight: bold;
    color:rgba(64,76,93);
    font-size: 15px;

}
.footer-head>.center-wrapper>.sub-footer-head2>p{
    margin: 0;
    padding: 0;
    font-weight: bold;
    color:rgba(64,76,93);
    font-size: 15px;

}
.footer-head>.center-wrapper>.sub-footer-head3>p{
    margin: 0;
    padding: 0;
    font-weight: bold;
    color:rgba(64,76,93);
    font-size: 15px;

}
.center-wrapper{
    width:35%;
    height:60px;
    display: flex;
    align-items: center;
    justify-content: center;

}
.sub-footer-head2{
    height:60px;
    width:50%;
    display:flex;
    justify-content: center;
    align-items: center;

}
.sub-footer-head3{
    height:60px;
    width:50%;
    display:flex;
    justify-content: center;
    align-items: center;


}

.sub-sub-footer-block{
    display:flex;
    justify-content:flex-start  ;
    align-items: center;
    width:60%;
    padding-left: 20px;
}

.switch-icon-footer-block{
    width:40%;
    display: flex;
    align-items: center;
    justify-content: space-around;


}
.switch-icon-footer-block>label{
    margin-top:16px;

}

.sub-language-block>.sub-sub-footer-block>p{
    margin: 0;
    padding: 0;
    font-weight: bold;
    color:rgba(64,76,93);
    font-size: 15px;

}
.contact-us{
    margin-top:50px;
    width:35%;
    height:400px;
    background-color:rgba(248,248,248);
    min-width: 400px;

}

.laguages-block{
    height:auto;
    width:400px;

}

.language-head{
    background-color:rgba(242, 242, 242);
    height:60px;
    display:flex;
    justify-content: space-between;
    align-items: center;
}
.sub-language-head{
    height:60px;
    width:40%;
    display:flex;
    justify-content: center;
    align-items: center;
}

.language-head>.sub-language-head>p{
    margin: 0;
    padding: 0;
    font-weight: bold;
    color:rgba(64,76,93);
    font-size: 15px;

}
.sub-language-block>.sub-sub-language-block>p{
    margin: 0;
    padding: 0;
    font-weight: bold;
    color:rgba(64,76,93);
    font-size: 15px;

}

.sub-language-block{
    height:60px;
    border-right:#c8c7c7 solid 1px;
    border-bottom:#c8c7c7 solid 1px;
    border-left:#c8c7c7 solid 1px;
    color:rgba(64,76,93);
    display:flex;
    justify-content: space-between;
    align-items: center;
}

.sub-sub-language-block{
    display:flex;
    justify-content: center;
    align-items: center;
    width:40%;
}

.switch-icon-language-block{
    width:90%;
    display: flex;
    align-items: center;
    justify-content: space-around;
}
.switch-icon-language-block>label{
    margin-top:16px;

}
.custom-switch>.react-switch-bg{
    border:1px rgba(64,76,93) solid;

}
div.sub-language-block:hover{
    border: none;
    box-shadow:0px 0px 8px 0px rgba(0, 0, 0, 0.3);
}
.contact-us-a{
    margin-top:50px;
    width:35%;
    height:400px;
    background-color:rgba(248,248,248);
    min-width: 400px;

}


.textLabelfoot {
    width: 90%;
    height: 80px;
    display: flex;
    justify-content: center;
    flex-direction: column;

}
.input-wrapper{
    width: 100%;
    height:350px;
    display: flex;
    flex-direction: column;
    padding-top: 20px;
    align-items: center;
}
.input-wrapper-a{
    width: 100%;
    height:500px;
    display: flex;
    flex-direction: column;
    padding-top: 20px;
    align-items: center;
}
.const-wrap {
    width: 60%;

    display: flex;
    flex-direction: column;
    flex-wrap: wrap;

    z-index: 1;
}

.new-article-cancel-alert {
    height: 250px;
    width: 400px;
    background: white;
    display: flex;
    flex-direction: column;
    justify-content: space-between;

}

.new-article-cancel-alert-text {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    width: 100%;
    height: 70%;
    border-bottom: solid #ccc 1px;
    padding: 0 50px;
}

.new-article-cancel-alert-btn {
    width: 100%;
    height: 30%;
    display: flex;
    justify-content: flex-end;
    align-items: center;
    padding: 0 50px;
}

.cancel-btn {
    border: none;
    background: white;
    color: #b21f2d;
}

.save-btn {
    border: none;
    color: white;
    background: #b21f2d;
    width: 100px;
    height: 40px;
    margin-left: 30px;
}

.cancel-box {
    position: fixed;
    left: 0;
    top: 0;
    width: 100%;
    height: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
    transition: background-color 400ms;
    background-color: rgba(0, 0, 0, 0.5);
    z-index: 10;
}

.main-a-const {

    margin-top: 100px;
    width: 100%;
    height: 1000px;

}

.breakdown-const {
    width: 100%;
    height: 300px;
}

.photo-of-the-day-const {
    width: 100%;
    height: 1300px;
    display: flex;
    flex-wrap: wrap;

}

.mots-sec-const {
    width: 100%;
    height: 300px;
    display: flex;
    flex-wrap: wrap;
    align-items: center;
    justify-content: space-between;
}

.const-wrapper {
    display: flex;
    flex-wrap: wrap;
    width: 100%;
    height: 250px;
    align-items: center;
    justify-content: space-between;
}

.txt-const {
    color: grey;
}

.small-select {
    width: 30%;
    height: 100px;
}

.big-select {
    width: 100%;
    height: 100px;
}

.previewComponent {
    width: 100%;
    height: 700px;
    display: flex;
    justify-content: center;
    align-items: center;
}

.imgPreview {
    width: 100%;
    height: 100%;
    position: absolute;
    top: 0;
    bottom: 0;

    display: flex;
    align-items: center;
    justify-content: center;
    background-color: rgb(246, 246, 246);
    border: 2px dotted #bebebe;
    border-radius: 10px;
}

input[type="file"] {
    position: absolute;
    left: 0;
    opacity: 0;
    top: 0;
    bottom: 0;
    width: 100%;
}
.picture{
    height: 400px;
}
.customPhotoupload {
    width: 100%;
    height: 100%;
    display: inline-block;
    position: relative;
}

.previewText {
    display: flex;
    justify-content: center;
    flex-wrap: wrap;
    width: 450px;
    font-size: 18px;
}

.formSelect {
    justify-content: space-between;
    display: flex;
    flex-direction: row;
    width: 100%;
}

.textForm {
    width: 100%;

    display: flex;
    flex-wrap: wrap;
    margin-left: 20%;
    flex-direction: column;
}

.textLabel {
    width: 100%;
    height: 125px;
    display: flex;
    justify-content: center;
    flex-direction: column;
}

.textInput {
    width: 100%;
    height: 50px;
}

.most-pop {
    width: 47%;
    height: 50px;
    background: rgb(246, 246, 246);
    display: flex;
    align-items: center;
    justify-content: space-between;
    padding-left: 5px;
}
