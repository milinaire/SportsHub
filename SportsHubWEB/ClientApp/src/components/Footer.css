.footer {
  box-shadow: 0 .25rem .75rem rgba(0, 0, 0, .05);
    text-align: center;
    clear: both;
  margin-bottom: 0;
}