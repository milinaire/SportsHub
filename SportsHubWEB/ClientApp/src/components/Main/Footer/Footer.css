.footer-categiries {
  clear: both;
  margin-bottom: 0;
  border-top: 3px solid #ccc;
  display: flex;
  justify-content: space-between;
  padding: 40px 200px 40px 20px;
}
.footer-categiry {
  width: 33%;
}
.footer-categiry h4 {
  padding-bottom: 25px;
}

.footer-info {
  clear: both;
  margin-bottom: 0;
  height: 100px;
  background-color: #d72130 !important;
  display: flex;
  align-items: center;
  justify-content: space-between;
}

.logo {
  font-size: 22px;
  width: 16rem;
  display: inline-block;
  height: 100%;
}
.logo a {
  text-decoration: none;
}
.logo-text {
  text-align: center;
  color: white;
}
.logo-text p {
  padding: 0;
  margin: 0;
  color: white !important;
}
.footer-link {
  height: 100%;
  width: 265px;
  display: flex;
  align-items: center;
  justify-content: center;
}
.copirating {
  display: inline-block;
  padding: auto;
  color: white;
  margin-right: 50px;
  text-align: right;
}
.copirating p {
  margin: 0;
}
.copirating a {
  text-decoration: none;
  color: white;
}
.input-email {
  border-radius: 0;
  border: 3px solid #ccc;
  padding: 5px;
}
.submit-email {
  border-radius: 0;
  border: 3px solid #d72130;
  color: white;
  background: #d72130;
  padding: 5px;
}
.footer-categiry p {
  color: #aaa;
}
