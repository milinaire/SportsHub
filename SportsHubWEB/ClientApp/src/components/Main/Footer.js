import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import './Footer.css';
import { Container } from "reactstrap";

export class Footer extends Component {




  render () {
    return (
      <footer>

          <div className={"footer"}>
              i am not a very beautiful footer. hi
          </div>

      </footer>
    );
  }
}
