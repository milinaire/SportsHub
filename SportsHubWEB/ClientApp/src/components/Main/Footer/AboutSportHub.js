import React, { Component, Fragment } from "react";
import "./Footer.css";
import {BaseLayout} from "../Layout/BaseLayout";

export class AboutSportHub extends Component {
  state = {
    Headline: "some headline from db",
    Text: "text from db",
  };
  render() {
    return (
      <Fragment>
        <BaseLayout>
          <div>
            <h1>{this.state.Headline}</h1>
            <p>{this.state.Text}</p>
          </div>
        </BaseLayout>
      </Fragment>

    );
  }
}
