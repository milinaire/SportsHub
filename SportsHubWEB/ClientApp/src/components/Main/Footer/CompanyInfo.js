import React, { Component, Fragment } from "react";
import "./Footer.css";
import {BaseLayout} from "../Layout/BaseLayout";

export class CompanyInfo extends Component {
  state = {
    Headline: "some headline from db",
    Text: "text from db",
  };
  render() {
    return (
      <Fragment>
        <BaseLayout>
          <div>
            {this.props.match.params.name}
            <h1>{this.state.Headline}</h1>
            <p>{this.state.Text}</p>
          </div>
        </BaseLayout>
      </Fragment>

    );
  }
}
