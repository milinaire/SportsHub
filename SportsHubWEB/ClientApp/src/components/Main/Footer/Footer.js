import React, { Component } from "react";
import { Link } from "react-router-dom";
import "./Footer.css";

export class Footer extends Component {
  state = {
    ContactUs: { HeadLine: "Contact Us", show: true, url: "contact" },
    AboutSportHub: { HeadLine: "About SportHub", show: true, url: "about" },
    CompanyInfo: {
      title: "COMPANY INFO",
      show: true,
      sections: [
        { HeadLine: "headline", show: true, url: "1" },
        { HeadLine: "headline", show: true, url: "2" },
        { HeadLine: "headline", show: true, url: "3" },
      ],
    },
    Contributors: {
      title: "CONTRIBUTORS",
      show: true,
      contributors: [
        { HeadLine: "headline", show: true, url: "1" },
        { HeadLine: "headline", show: true, url: "2" },
        { HeadLine: "headline", show: true, url: "3" },
      ],
    },
    NewsLetter: {
      title: "NEWSLETTER",
      show: true,
      showDescription: true,
    },
  };
  render() {
    return (
      <footer>
        <div className={"footer-categiries"}>
          {this.state.CompanyInfo.show ? (
            <div className="footer-categiry">
              <h4>
                <b>{this.state.CompanyInfo.title}</b>
              </h4>
              {this.state.AboutSportHub.show ? (
                <Link to={`/${this.state.AboutSportHub.url}`}>
                  <p>{this.state.AboutSportHub.HeadLine}</p>
                </Link>
              ) : null}
              {this.state.CompanyInfo.sections.map((section) =>
                section.show ? (
                  <Link to={`/companyinfo/${section.url}`}>
                    <p>{section.HeadLine}</p>
                  </Link>
                ) : null
              )}
              {this.state.ContactUs.show ? (
                <Link to={`/${this.state.ContactUs.url}`}>
                  <p>{this.state.ContactUs.HeadLine}</p>
                </Link>
              ) : null}
            </div>
          ) : null}
          {this.state.Contributors.show ? (
            <div className="footer-categiry">
              <h4>
                <b>{this.state.Contributors.title}</b>
              </h4>
              {this.state.Contributors.contributors.map((contributor) =>
                contributor.show ? (
                  <Link to={`/contributors/${contributor.url}`}>
                    <p>{contributor.HeadLine}</p>
                  </Link>
                ) : null
              )}
            </div>
          ) : null}
          {this.state.NewsLetter.show ? (
            <div className="footer-categiry">
              <h4>
                <b>{this.state.NewsLetter.title}</b>
              </h4>
              {this.state.NewsLetter.showDescription ? (
                <h6>Sign up to receive the latest sport news</h6>
              ) : null}
              <form>
                <input
                  className="input-email"
                  type="email"
                  name="email"
                  placeholder="Your email address"
                />
                <input
                  className="submit-email"
                  type="button"
                  value="Subscribe"
                />
              </form>
            </div>
          ) : null}
        </div>
        <div className={"footer-info"}>
          <div className="logo">
            <a href="/">
              <div className="footer-link">
                <div className="logo-text">
                  <a href="/">
                    <p>Sports Hub</p>
                  </a>
                </div>
              </div>
            </a>
          </div>
          <div className="copirating">
            <p>Copirating c 2019 SportHub</p>
            <Link to="/privacy">Privacy</Link>/<a href="/terms">Terms</a>
          </div>
        </div>
      </footer>
    );
  }
}
