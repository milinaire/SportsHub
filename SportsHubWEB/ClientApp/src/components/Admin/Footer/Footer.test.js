import React from 'react';
import ReactDOM from 'react-dom';
import {MemoryRouter} from 'react-router-dom';

import renderer from 'react-test-renderer';
import Footer from "./Footer";


describe('my beverage', () => {
  const props = {
    mainArticles:{mainArticles:[]}
  }
  test('renders without crashing', async () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <MemoryRouter>
        <Footer {...props}/>
      </MemoryRouter>, div);

  })
  test('',()=>{
    const tree = renderer
      .create(<Footer {...props}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})