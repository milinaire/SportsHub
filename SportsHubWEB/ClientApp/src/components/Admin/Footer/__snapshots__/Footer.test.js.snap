// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my beverage  1`] = `
<div
  className="admin-footer"
>
  <div
    className="admin-footer-header"
  >
    <a
      onClick={[Function]}
    >
         COMPANY INFO 
    </a>
    <a
      onClick={[Function]}
    >
        CONTRIBUTORS 
    </a>
    <a
      onClick={[Function]}
    >
        NEWSLETTER 
    </a>
  </div>
  <div
    className="footer-wrapper"
  >
    <div
      className="footer-block"
    >
      <div
        className="footer-head"
      >
        <div
          className="sub-footer-head1"
        >
          <p>
            FOOTER MENU NAMES
          </p>
        </div>
        <div
          className="center-wrapper"
        >
          <div
            className="sub-footer-head2"
          >
            <p>
              HIDE/SHOW
            </p>
          </div>
          <div
            className="sub-footer-head3"
          >
            <p>
              ACTIONS
            </p>
          </div>
        </div>
      </div>
      <div
        className="sub-language-block"
      >
        <div
          className="sub-sub-footer-block"
          onClick={[Function]}
        >
          <p>
            ABOUT SPORTS HUB
          </p>
        </div>
        <div
          className="switch-icon-footer-block"
        >
          <label>
            <div
              className="custom-switch"
              style={
                Object {
                  "MozTransition": "opacity 0.25s",
                  "MozUserSelect": "none",
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "WebkitTransition": "opacity 0.25s",
                  "WebkitUserSelect": "none",
                  "borderRadius": 10,
                  "direction": "ltr",
                  "display": "inline-block",
                  "msUserSelect": "none",
                  "opacity": 1,
                  "position": "relative",
                  "textAlign": "left",
                  "touchAction": "none",
                  "transition": "opacity 0.25s",
                  "userSelect": "none",
                }
              }
            >
              <div
                className="react-switch-bg"
                onClick={[Function]}
                onMouseDown={[Function]}
                style={
                  Object {
                    "MozTransition": "background 0.25s",
                    "WebkitTransition": "background 0.25s",
                    "background": "#rreeyy",
                    "borderRadius": 10,
                    "cursor": "pointer",
                    "height": 20,
                    "margin": 0,
                    "position": "relative",
                    "transition": "background 0.25s",
                    "width": 40,
                  }
                }
              />
              <div
                className="react-switch-handle"
                onClick={[Function]}
                onMouseDown={[Function]}
                onTouchCancel={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "MozTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "WebkitTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "background": "#F0F0F0",
                    "border": 0,
                    "borderRadius": "50%",
                    "boxShadow": "0 0 8px 0 rgba(0, 0, 0, 0.3)",
                    "cursor": "pointer",
                    "display": "inline-block",
                    "height": 15,
                    "outline": 0,
                    "position": "absolute",
                    "top": 2.5,
                    "transform": "translateX(2.5px)",
                    "transition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "width": 15,
                  }
                }
              />
              <input
                aria-checked={false}
                checked={false}
                disabled={false}
                name="check1"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyUp={[Function]}
                role="switch"
                style={
                  Object {
                    "border": 0,
                    "clip": "rect(0 0 0 0)",
                    "height": 1,
                    "margin": -1,
                    "overflow": "hidden",
                    "padding": 0,
                    "position": "absolute",
                    "width": 1,
                  }
                }
                type="checkbox"
              />
            </div>
          </label>
          FaTrash/&gt;
        </div>
      </div>
      <div
        className="sub-language-block"
      >
        <div
          className="sub-sub-footer-block"
        >
          <p>
            News / In the Press
          </p>
        </div>
        <div
          className="switch-icon-footer-block"
        >
          <label>
            <div
              className="custom-switch"
              style={
                Object {
                  "MozTransition": "opacity 0.25s",
                  "MozUserSelect": "none",
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "WebkitTransition": "opacity 0.25s",
                  "WebkitUserSelect": "none",
                  "borderRadius": 10,
                  "direction": "ltr",
                  "display": "inline-block",
                  "msUserSelect": "none",
                  "opacity": 1,
                  "position": "relative",
                  "textAlign": "left",
                  "touchAction": "none",
                  "transition": "opacity 0.25s",
                  "userSelect": "none",
                }
              }
            >
              <div
                className="react-switch-bg"
                onClick={[Function]}
                onMouseDown={[Function]}
                style={
                  Object {
                    "MozTransition": "background 0.25s",
                    "WebkitTransition": "background 0.25s",
                    "background": "#rreeyy",
                    "borderRadius": 10,
                    "cursor": "pointer",
                    "height": 20,
                    "margin": 0,
                    "position": "relative",
                    "transition": "background 0.25s",
                    "width": 40,
                  }
                }
              />
              <div
                className="react-switch-handle"
                onClick={[Function]}
                onMouseDown={[Function]}
                onTouchCancel={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "MozTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "WebkitTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "background": "#F0F0F0",
                    "border": 0,
                    "borderRadius": "50%",
                    "boxShadow": "0 0 8px 0 rgba(0, 0, 0, 0.3)",
                    "cursor": "pointer",
                    "display": "inline-block",
                    "height": 15,
                    "outline": 0,
                    "position": "absolute",
                    "top": 2.5,
                    "transform": "translateX(2.5px)",
                    "transition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "width": 15,
                  }
                }
              />
              <input
                aria-checked={false}
                checked={false}
                disabled={false}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyUp={[Function]}
                role="switch"
                style={
                  Object {
                    "border": 0,
                    "clip": "rect(0 0 0 0)",
                    "height": 1,
                    "margin": -1,
                    "overflow": "hidden",
                    "padding": 0,
                    "position": "absolute",
                    "width": 1,
                  }
                }
                type="checkbox"
              />
            </div>
          </label>
          FaTrash/&gt;
        </div>
      </div>
      <div
        className="sub-language-block"
      >
        <div
          className="sub-sub-footer-block"
        >
          <p>
            Advertising / Sports Blogger Ad Network
          </p>
        </div>
        <div
          className="switch-icon-footer-block"
        >
          <label>
            <div
              className="custom-switch"
              style={
                Object {
                  "MozTransition": "opacity 0.25s",
                  "MozUserSelect": "none",
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "WebkitTransition": "opacity 0.25s",
                  "WebkitUserSelect": "none",
                  "borderRadius": 10,
                  "direction": "ltr",
                  "display": "inline-block",
                  "msUserSelect": "none",
                  "opacity": 1,
                  "position": "relative",
                  "textAlign": "left",
                  "touchAction": "none",
                  "transition": "opacity 0.25s",
                  "userSelect": "none",
                }
              }
            >
              <div
                className="react-switch-bg"
                onClick={[Function]}
                onMouseDown={[Function]}
                style={
                  Object {
                    "MozTransition": "background 0.25s",
                    "WebkitTransition": "background 0.25s",
                    "background": "#rreeyy",
                    "borderRadius": 10,
                    "cursor": "pointer",
                    "height": 20,
                    "margin": 0,
                    "position": "relative",
                    "transition": "background 0.25s",
                    "width": 40,
                  }
                }
              />
              <div
                className="react-switch-handle"
                onClick={[Function]}
                onMouseDown={[Function]}
                onTouchCancel={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "MozTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "WebkitTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "background": "#F0F0F0",
                    "border": 0,
                    "borderRadius": "50%",
                    "boxShadow": "0 0 8px 0 rgba(0, 0, 0, 0.3)",
                    "cursor": "pointer",
                    "display": "inline-block",
                    "height": 15,
                    "outline": 0,
                    "position": "absolute",
                    "top": 2.5,
                    "transform": "translateX(2.5px)",
                    "transition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "width": 15,
                  }
                }
              />
              <input
                aria-checked={false}
                checked={false}
                disabled={false}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyUp={[Function]}
                role="switch"
                style={
                  Object {
                    "border": 0,
                    "clip": "rect(0 0 0 0)",
                    "height": 1,
                    "margin": -1,
                    "overflow": "hidden",
                    "padding": 0,
                    "position": "absolute",
                    "width": 1,
                  }
                }
                type="checkbox"
              />
            </div>
          </label>
          FaTrash/&gt;
        </div>
      </div>
      <div
        className="sub-language-block"
      >
        <div
          className="sub-sub-footer-block"
        >
          <p>
            Events
          </p>
        </div>
        <div
          className="switch-icon-footer-block"
        >
          <label>
            <div
              className="custom-switch"
              style={
                Object {
                  "MozTransition": "opacity 0.25s",
                  "MozUserSelect": "none",
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "WebkitTransition": "opacity 0.25s",
                  "WebkitUserSelect": "none",
                  "borderRadius": 10,
                  "direction": "ltr",
                  "display": "inline-block",
                  "msUserSelect": "none",
                  "opacity": 1,
                  "position": "relative",
                  "textAlign": "left",
                  "touchAction": "none",
                  "transition": "opacity 0.25s",
                  "userSelect": "none",
                }
              }
            >
              <div
                className="react-switch-bg"
                onClick={[Function]}
                onMouseDown={[Function]}
                style={
                  Object {
                    "MozTransition": "background 0.25s",
                    "WebkitTransition": "background 0.25s",
                    "background": "#rreeyy",
                    "borderRadius": 10,
                    "cursor": "pointer",
                    "height": 20,
                    "margin": 0,
                    "position": "relative",
                    "transition": "background 0.25s",
                    "width": 40,
                  }
                }
              />
              <div
                className="react-switch-handle"
                onClick={[Function]}
                onMouseDown={[Function]}
                onTouchCancel={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "MozTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "WebkitTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "background": "#F0F0F0",
                    "border": 0,
                    "borderRadius": "50%",
                    "boxShadow": "0 0 8px 0 rgba(0, 0, 0, 0.3)",
                    "cursor": "pointer",
                    "display": "inline-block",
                    "height": 15,
                    "outline": 0,
                    "position": "absolute",
                    "top": 2.5,
                    "transform": "translateX(2.5px)",
                    "transition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "width": 15,
                  }
                }
              />
              <input
                aria-checked={false}
                checked={false}
                disabled={false}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyUp={[Function]}
                role="switch"
                style={
                  Object {
                    "border": 0,
                    "clip": "rect(0 0 0 0)",
                    "height": 1,
                    "margin": -1,
                    "overflow": "hidden",
                    "padding": 0,
                    "position": "absolute",
                    "width": 1,
                  }
                }
                type="checkbox"
              />
            </div>
          </label>
          FaTrash/&gt;
        </div>
      </div>
      <div
        className="sub-language-block"
      >
        <div
          className="sub-sub-footer-block"
          onClick={[Function]}
        >
          <p>
            Contact Us
          </p>
        </div>
        <div
          className="switch-icon-footer-block"
        >
          <label>
            <div
              className="custom-switch"
              style={
                Object {
                  "MozTransition": "opacity 0.25s",
                  "MozUserSelect": "none",
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "WebkitTransition": "opacity 0.25s",
                  "WebkitUserSelect": "none",
                  "borderRadius": 10,
                  "direction": "ltr",
                  "display": "inline-block",
                  "msUserSelect": "none",
                  "opacity": 1,
                  "position": "relative",
                  "textAlign": "left",
                  "touchAction": "none",
                  "transition": "opacity 0.25s",
                  "userSelect": "none",
                }
              }
            >
              <div
                className="react-switch-bg"
                onClick={[Function]}
                onMouseDown={[Function]}
                style={
                  Object {
                    "MozTransition": "background 0.25s",
                    "WebkitTransition": "background 0.25s",
                    "background": "#rreeyy",
                    "borderRadius": 10,
                    "cursor": "pointer",
                    "height": 20,
                    "margin": 0,
                    "position": "relative",
                    "transition": "background 0.25s",
                    "width": 40,
                  }
                }
              />
              <div
                className="react-switch-handle"
                onClick={[Function]}
                onMouseDown={[Function]}
                onTouchCancel={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "MozTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "WebkitTransition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "background": "#F0F0F0",
                    "border": 0,
                    "borderRadius": "50%",
                    "boxShadow": "0 0 8px 0 rgba(0, 0, 0, 0.3)",
                    "cursor": "pointer",
                    "display": "inline-block",
                    "height": 15,
                    "outline": 0,
                    "position": "absolute",
                    "top": 2.5,
                    "transform": "translateX(2.5px)",
                    "transition": "background-color 0.25s, transform 0.25s, box-shadow 0.15s",
                    "width": 15,
                  }
                }
              />
              <input
                aria-checked={false}
                checked={false}
                disabled={false}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyUp={[Function]}
                role="switch"
                style={
                  Object {
                    "border": 0,
                    "clip": "rect(0 0 0 0)",
                    "height": 1,
                    "margin": -1,
                    "overflow": "hidden",
                    "padding": 0,
                    "position": "absolute",
                    "width": 1,
                  }
                }
                type="checkbox"
              />
            </div>
          </label>
          FaTrash/&gt;
        </div>
      </div>
    </div>
    <div
      className="contact-us-a"
    >
      <div
        className="footer-head"
      >
        <div
          className="sub-footer-head1"
        >
          <p>
            ABOUT SPORTS HUB
          </p>
        </div>
        <div
          className="center-wrapper"
        >
          <div
            className="sub-footer-head2"
          >
            <p />
          </div>
          <div
            className="sub-footer-head3"
          >
            <p />
          </div>
        </div>
      </div>
      <div
        className="input-wrapper-a"
      >
        <label
          className="textLabelfoot"
        >
          <p
            className="txt-const"
          >
            HEADLINE
          </p>
          <input
            className="textInput"
            name="HeadLine"
            type="text"
          />
        </label>
      </div>
    </div>
    <div
      className="footer-block1"
    >
      <div
        className="footer-head"
      >
        <div
          className="sub-footer-head1"
        >
          <p>
            PRIVACY AND TERMS
          </p>
        </div>
        <div
          className="center-wrapper"
        >
          <div
            className="sub-footer-head2"
          >
            <p />
          </div>
          <div
            className="sub-footer-head3"
          >
            <p />
          </div>
        </div>
      </div>
      <div
        className="sub-language-block"
      >
        <div
          className="sub-sub-footer-block"
          onClick={[Function]}
        >
          <p>
            Privacy policy
          </p>
        </div>
      </div>
      <div
        className="sub-language-block"
      >
        <div
          className="sub-sub-footer-block"
        >
          <p>
            Terms and contidions
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
